hotswap_rgb/README.md

```diff
@@ -0,0 +1,56 @@
+# dumbpad - Hotswap - RGB (aka v3x)
+
+Designed by [deveth0](https://www.github.com/deveth0).
+
+## Description
+
+Version of the original dumbpad that features Kailh Hotswap Sockets and (optional) RGB per-key lightning.
+
+### Revisions
+
+- 3.1 Initial published PCB
+
+
+#### dumbpad v3.1
+
+![pcbs](dumbpad_v3_1_pcbs.jpg)
+![finished](dumbpad_v3_1_finished.jpg)
+
+
+#### Renders
+![front](dumbpad_v3_1_front.png)
+
+![back](dumbpad_v3_1_back.png)
+
+
+### Parts
+
+* 16x Kailh Hotswap Sockets (rev 2)
+* 16x Cherry-style mechanical switches
+* 17x 1n4148 diodes (thru hole)
+* 1x Arduino Pro Micro or pin-compatible ATmega32u4-based MCU
+* 1x EC11 rotary encoder with pushbutton (7-pin)
+* (optional) 16x SK6812 mini e LEDs
+* (optional) 1x 6mm tactile switch (to reset MCU)
+* (optional) 3x 3mm LEDs (whatever color)
+* (optional) 3x 330ohm or similar resistors for the LEDS (not needed if not adding LEDs)
+
+
+## Making the PCB
+
+Submit either the KiCad `.kicad_pcb` or the gerber files to your prefered PCB manufacturer.
+
+### Notes on soldering
+
+It's recommended to start with the Sockets and then add the remaining parts.
+
+Solder the SK6812 LEDs with the missing corners aligned with the corners printed on the PCB as shown in the image.
+
+*NOTE:* the orientation of the LEDs changes between the lines!
```

Comment:
Can you replace the photo with one that shows LEDs on rows with different orientations? Specifically, unsoldered LEDs placed in the proper orientation.

Maybe even a big red circle around the chamfered edge :P

Reply:
something like this? https://github.com/deveth0/dumbpad/blob/master/hotswap_rgb/img/dumbpad_v3_1_rgb_mounting.jpg

Reply:
perfect!